app/helpers/users_helper.rb

```diff
@@ -16,14 +16,7 @@ module UsersHelper
   #   gravatar.
   #
   def gravatar_for(user, options = {})
-    options = {
-      size: 48
-    }.merge(options)
-
-    size = options[:size]
-    gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
-    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
-    image_tag(gravatar_url, alt: user.name, class: 'gravatar')
+    ENV['DISABLE_GRAVATAR'].present? ? no_gravatar_image : gravatar_image(user, options)
```

Comment:
That's a good idea...though I'm not sure I full understand how Rollout works.  I know we define the rollout features in a databag in the community cookbook, where would a user define the rollout features in private Supermarket?